src/include/duckdb/common/types/hugeint.hpp

```diff
@@ -19,17 +19,31 @@ class Hugeint {
 public:
 	//! Convert a string to a hugeint object
 	static bool FromString(string str, hugeint_t &result);
+	//! Convert a uuid string to a hugeint object
+	static bool FromUUIDString(string str, hugeint_t &result);
```

Comment:
Can all the UUID functions in `Hugeint` be moved to a separate `UUID` class (in `duckdb/common/types/uuid.hpp`)?